vllm/outputs.py

```diff
@@ -146,3 +165,58 @@ def __repr__(self) -> str:
                 f"finished={self.finished}, "
                 f"metrics={self.metrics}, "
                 f"lora_request={self.lora_request})")
+
+
+class EmbeddingRequestOutput:
```

Comment:
I think either we need to incorporate this EmbeddingRequestOutput inside RequestOutput or we rename the original RequestOutput to be CompletionRequestOutput, then change the name for all the other places. To me, this EmbeddingRequestOutput should be a child class of RequestOutput instead of a parallel one.

Reply:
Done.